app/appserver/static/js/handle_autoforward.js

```diff
@@ -1,47 +1,69 @@
-require([
-     "splunkjs/mvc",
-     ], function(mvc) {
- mvc.setFilter("dbquote", function(inputValue) {
-     return inputValue.replace(/"/g, '\\"').replace(/'/g, "\\'");
-   });
- });
+require(["splunkjs/mvc/utils"], function (SplunkUtils) {
+    var app_name = "TA-Databricks"
+    require.config({
+        paths: {
+            'jquery_databricks': '../app/' + app_name + '/js/jquery_databricks',
```

Comment:
Yes we need the bundled jquery for $("#retryButton").click, because it ensures that jquery 3.5.0 is used. If not,  depending on the slunk version used, an older version of jquery may be used which will cause failure in Splunk appinspect report.

Reply:
do we **seriously** need jQuery (a library of 300kb) just to do `document.getElementById('retryButton').onClick(`?

Reply:
@nfx 
In the launch notebook dashboard, a retry button is provided in the panel when the notebook run fails due to some reason(cluster is not running etc). However when the dashboard is opened for the first time or the notebooks are run successfully, the retry button is not available. So in this case , document.getElementById('retryButton').onClick( doesn't work as expected. Hence we need to use jquery .

![image](https://user-images.githubusercontent.com/99265655/159740850-8e97c093-fcbc-4826-8994-e7f8c82e8fab.png)

Reply:
Removed jquery